### ISSUE TITLE
UPDATE FROM: This Works on SQL-Server but not not on Oracle:

### ISSUE THREAD

**Issue by MaceWindu** [linq2db/linq2db#2330]
This Works on SQL-Server but not not on Oracle:
```cs
                var q = from w in objDbContext.W_DOWNLOAD_VTB
                        join b in objDbContext.VERSICHERTE_BETRIEB on new { w.BETRIEB_ID, w.VERSINT_ID } equals new { b.BETRIEB_ID, b.VERSINT_ID }
                        where w.REPORT_ID == lngReportId
                           && b.GUELTIG_AB == (from b2 in objDbContext.VERSICHERTE_BETRIEB
                                               where b2.VERSINT_ID == b.VERSINT_ID
                                                  && b2.BETRIEB_ID == b.BETRIEB_ID
                                               select b2.GUELTIG_AB).Max()
                        select new { w, b };

                q.Update(objDbContext.W_DOWNLOAD_VTB, obj => new Type
                {
                    EINTRITT_BETRIEB = obj.b.GUELTIG_AB
                });
```

_Originally posted by @BenoitStutz in https://github.com/linq2db/linq2db/pull/1278#issuecomment-655440028_

**Comment by BenoitStutz** [comment]
It works with Version 3.1.0

**Comment by sdanyliv** [comment]
So, then we were bad in googling.

**Comment by MaceWindu** [comment]
Was fixed in 3.2.0 release
